db/__init__.py

```diff
@@ -71,6 +72,8 @@ def _create_engine():
         uri,
         # Set echo=True for full query logging:
         echo=False,
+        json_serializer=lambda obj: json.dumps(obj, use_decimal=True),
+        json_deserializer=lambda obj: json.loads(obj, use_decimal=True),
```

Comment:
The built-in JSON doesn't seem to be able to handle decimals. Need to be careful to always use this one across the codebase.